cmd/nodecmd/validateSubnet.go

```diff
@@ -184,17 +172,22 @@ func validateSubnet(_ *cobra.Command, args []string) error {
 			continue
 		}
 		// we have to check if node is synced to subnet before adding the node as a validator
-		isSubnetSynced, err := getNodeSubnetSyncStatus(blockchainID.String(), clusterName, host, false)
+		subnetSyncStatus, err := getNodeSubnetSyncStatus(blockchainID.String(), clusterName, host)
 		if err != nil {
 			ux.Logger.PrintToUser(fmt.Sprintf("Failed to get subnet sync status for node %s", host))
 			failedNodes = append(failedNodes, host)
 			nodeErrors = append(nodeErrors, err)
 			continue
 		}
-		if !isSubnetSynced {
-			ux.Logger.PrintToUser(fmt.Sprintf("Failed to add node %s as subnet validator as node is not synced to subnet yet", host))
+		if subnetSyncStatus != status.Syncing.String() {
 			failedNodes = append(failedNodes, host)
-			nodeErrors = append(nodeErrors, errors.New("node is not synced to subnet yet, please try again later"))
+			if subnetSyncStatus == status.Validating.String() {
+				ux.Logger.PrintToUser(fmt.Sprintf("Failed to add node %s as subnet validator as node is already a subnet validator", host))
```

Comment:
I believe PrintToUser already accept format strings and format values. No need for Sprintf